src/courses/beginner/03.md

```diff
@@ -165,7 +163,7 @@ e.g. `its("signal") { should eq "on" }`
 
 ### Comprehension Check!
 
-Look at the 4 file structures below. Determine which are valid InSpec Profiles!
+Look at the 4 file structures below. Determine which are valid InSpec Profiles that will run the tests!
```

Comment:
`inspec exec` will not error out if the profile doesn't contain a `controls` dir - it will run and say 'No tests executed.'
Does this help clarify it?